service/controller/v5/resource/containerurl/update.go

```diff
@@ -0,0 +1,26 @@
+package containerurl
+
+import (
+	"context"
+
+	"github.com/giantswarm/azure-operator/service/controller/v5/key"
+	"github.com/giantswarm/microerror"
+)
+
+func (r *Resource) EnsureUpdated(ctx context.Context, obj interface{}) error {
```

Comment:
That method does not do anything. I would say we drop it. When you have the normal resource it is only `EnsureCreated` and `EnsureDeleted`. When the container URL is necessary for deletion, the latter should be sorted as well.